.circleci/continue_config.yml

```diff
@@ -346,10 +346,10 @@ workflows:
                 - bigquery
                 - clickhouse-cloud
                 - athena
-          filters:
-           branches:
-             only:
-               - main
+          #filters:
+          # branches:
+          #   only:
+          #     - main
```

Comment:
TODO: revert this. I just wanted to check that the cloud engines are running fine as well